featuretools/primitives/standard/transform_primitive.py

```diff
@@ -591,3 +592,34 @@ def generate_name(self, base_feature_names):
             name += u", unit={}".format(self.unit)
         name += u")"
         return name
+
+
+class Age(TransformPrimitive):
+    """Calculates the age in years as a floating point number given a
```

Comment:
My one comment is that it might be helpful to have an option to have ages return in integer form as well. However, this is pretty easy to do on the end of whoever is using it, but I think that is my only potential improvement I can think of right now.

Reply:
We could add it as an optional argument. Something like this work:
```python
class Age(TransformPrimitive):
    def __init__(self, decimals= None):
        self.decimals = decimals

    def get_function(self):
        def age(x, time=None):
            values_ = (time - x).dt.days / 365
            if self.decimals is not None:
                return np.around(values_, self.decimals)
        return age
```

However, I'm not the biggest fan. The end-user could round afterwards.

Reply:
Up to you, I agree that the user could round, but its pretty easy to add the optional argument and the optionality doesn't change the experience users who don't want to round.

Reply:
i think i'd stay away from adding this from now, otherwise we might have to add it to other primitives where people want rounding. let's get this merged in as is and we can always add it later if the need becomes strong